include/mull/Reporters/MutationTestingElementsReporter.h

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+#include "Reporter.h"
+
+#include "mull/MutationPoint.h"
+
+#include <clang/Basic/SourceLocation.h>
+
+#include <map>
+
+namespace mull {
+class MutationPoint;
+class MutationTestingElementsReporter : public Reporter {
+  const std::map<MutationPoint *, BeginEndPair> &mutationPointsRanges;
```

Comment:
Shouldn't a mutation point have two pairs? I.e. 'begin line/column' and 'end line/column'